src/newLaunch/launchConfig.ts

```diff
@@ -19,8 +18,8 @@ export const CategoryNames = [
 ];
 
 export class SocialLinkSpec {
-  public media: string;
-  public url: string;
+  public media: string
```

Comment:
please run `npm run lint`

Reply:
lint did not catch this missing `;` for some reason.